### PR TITLE
add the standard files

### PR DESCRIPTION
**Pull request by Emily-Jiang**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Contributing to Jakarta Concurrency
+
+Thanks for your interest in this project.
+
+## Project description
+
+Jakarta Concurrency provides a specification document, API and TCK for using concurrency from application components without compromising container integrity while still preserving the Jakarta EE platform's fundamental benefits.
```
Comment (pzygielo) [inline CONTRIBUTING.md:7]: :eyes:

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+ * https://projects.eclipse.org/projects/ee4j.cu
```
Comment (pzygielo) [inline CONTRIBUTING.md:9]: :eyes:

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+ * https://github.com/eclipse-ee4j/concurrency-ri
+
+## Eclipse Contributor Agreement
+
+Before your contribution can be accepted by the project team contributors must
+electronically sign the Eclipse Contributor Agreement (ECA).
+
+ * http://www.eclipse.org/legal/ECA.php
+
+Commits that are provided by non-committers must have a Signed-off-by field in
```
Comment (pzygielo) [inline CONTRIBUTING.md:30]: No.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+* https://eclipse.org/projects/dev_process
+* https://www.eclipse.org/org/documents/Eclipse_IP_Policy.pdf
+* https://jakarta.ee/about/jesp/
+* https://www.eclipse.org/legal/efsp_non_assert.php
+
+## Contact
+
+Contact the project developers via the project's "dev" list.
+
+ * https://accounts.eclipse.org/mailing-list/cu-dev
```
Comment (pzygielo) [inline CONTRIBUTING.md:58]: :eyes:

**Comment by Emily-Jiang** [comment]
@pzygielo thanks for reviewing  the files! I did the update but did not manage to push the saved version. These files should be up to date.

File: CONTRIBUTING.md (hunk 3 above)
Comment (Emily-Jiang) [reply CONTRIBUTING.md:29]: This is the standard process defined by Eclipse Foundatione and adopted by Jakarta EE specifications.

Comment (pzygielo) [reply CONTRIBUTING.md:29]: It's [no longer needed](https://www.eclipse.org/lists/eclipse.org-committers/msg01291.html) to sign off commits.

Comment (Emily-Jiang) [reply CONTRIBUTING.md]: Yes, you were right. I have removed the sentences. I guess the contribution.md from other Jakarta EE projects need to be updated to reflect the current process.

**Review by otaviojava — APPROVED** [review]

File: NOTICE.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
+# Notices for Jakarta Contexts and Dependency Injection
```
Comment (njr-11) [inline NOTICE.md:1]: Switch to Jakarta Data

**Pull request merged**